rewrite-maven/src/main/java/org/openrewrite/maven/tree/DependencyManagementDependency.java

```diff
@@ -87,6 +87,24 @@
             // FIXME should be active properties by profile as well? also parent properties?
             return maven.getProperties();
         }
+
+        @Override
+        public @Nullable String getClassifier() {
+            // classifier isn't possible on Imported
+            return null;
+        }
+
+        @Override
+        public @Nullable Scope getScope() {
+            // scope isn't possible on Imported
+            return null;
+        }
+
+        @Override
+        public Set<GroupArtifact> getExclusions() {
+            // exclusions aren't possible on Imported
+            return new HashSet<>();
+        }
```

Comment:
If half of the interface isn't actually applicable, is it really appropriate to say that `DependencyManagementDependency` is a `DependencyDescriptor` ?

Maybe it would be more appropriate to have a "GroupArtifactVersion" interface that `DependencyManagementDependency` and `DependencyDescriptor` both extend.

Reply:
There is evidence in `RawMavenResolver` that a group/artifact/version interface could be useful in other areas as well. 

https://github.com/openrewrite/rewrite/blob/554ccefa357c390d54662dbb0de9d760a9eb5496/rewrite-maven/src/main/java/org/openrewrite/maven/internal/RawMavenResolver.java#L634-L640